### DISCUSSION TITLE
--live option is not working with IntelliJ anymore

### DISCUSSION THREAD

**Discussion by ennishol** [jbangdev/jbang#1532] [Q&A]
I start editing from the command line with `jbang edit --live foo.java` and after adding a new dependency `//DEP` in IDE, I see the project is regenerated by jbang in the terminal,  but IntelliJ does not get the update. Is it known issue or does anyone know how to fix

I remember it used to work in older versions of jbang or ide. For now, I just re-run jbang command and it helps

**Comment by maxandersen** [comment]
does the build.gradle file not have the updated dependency? intellij does not pick up the changes without reloading the gradle file.

**Reply by ennishol** [reply]
@maxandersen  no. no dependency is added just: `dependencies {}` in build.gradle

**Comment by ennishol** [comment]
One more thing. jbang regenerates the project, but the new dependency is not fetched. Only after re-run. So, it looks like jbang does notice the changed file, but the change itself is not picked up

```
user@host ~/jbangs (master)$ jbang edit --live junk.java
[jbang] Starting 'idea'.
[jbang] Watching for changes in ~/jbangs
[jbang] Regenerating project.
[jbang] Regenerating project.
```

now run again
```
user@host ~/jbangs (master)$ jbang edit --live junk.java
[jbang] Resolving dependencies...
[jbang]    org.apache.commons:commons-collections4:4.4
[jbang] Dependencies resolved
[jbang] Starting 'idea'.
[jbang] Watching for changes in ~/jbangs
```

As you can see, only after re-running the jbang command the new dependency is resolved
